streams/service/src/main/java/com/hortonworks/streamline/streams/service/SchemaResource.java

```diff
@@ -190,4 +229,27 @@ public Response getSchemaForVersion(@PathParam("schemaName") String schemaName,
         }
     }
 
+    @GET
+    @Path("/{schemaName}/branches")
+    @Timed
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getSchemaBranches(@PathParam("schemaName") String schemaName,
+                                      @Context SecurityContext securityContext) {
+        try {
+            Collection<SchemaBranch> schemaBranches = schemaRegistryClient.getSchemaBranches(schemaName);
```

Comment:
Looks like we will always return MASTER branch. In that case may be make the schemaBranch UI field mandatory  and set defaultValue to MASTER ?